### ISSUE TITLE
404 The resource you were accessing could not be found

### ISSUE THREAD

**Issue by Aeron** [caddy-dns/digitalocean#1]
I’ve decided to give another try to Caddy 2 with DigitalOcean DNS challenge. And I found out it’s still unusable.

Built it via `caddy:2-builder` image without a problem. Did run it properly on my VPS and saw the following in logs:

```
2020/08/03 00:29:21 [INFO] [*.mydomain.com] acme: use dns-01 solver
2020/08/03 00:29:21 [INFO] [*.mydomain.com] acme: Preparing to solve DNS-01
2020/08/03 00:29:21 [INFO] [*.mydomain.com] acme: Cleaning DNS-01 challenge
2020/08/03 00:29:21 [WARN] [*.mydomain.com] acme: cleaning up failed: no memory of presenting a DNS record for mydomain.com 
2020/08/03 00:29:22 [ERROR] error: one or more domains had a problem:
[*.mydomain.com] [*.mydomain.com] acme: error presenting token: POST https://api.digitalocean.com/v2/domains/mydomain.com./records: 404 The resource you were accessing could not be found.
```

The problem here is `mydomain.com.` in the last record’s POST URL because [there must be no dot](https://developers.digitalocean.com/documentation/v2/#update-a-domain-record) in the end. But somehow it’s there. It’s the same with both my domains, yet they specified correctly in my Caddyfile, without a dot.

At this point, I’m not sure if it’s a problem with this code or [libdns/digitalocean](https://github.com/libdns/digitalocean) or [libdns/libdns](https://github.com/libdns/libdns) itself, I’ve inspected them only shallowly. But I’m pretty sure it’s one of these three.

**Comment by scoobybejesus** [comment]
Same issue for me.

**Comment by mtricht** [comment]
@Aeron can you give it another try? Should be fixed with #2

**Comment by Aeron** [comment]
> @Aeron can you give it another try? Should be fixed with #2

Yup, everything works now. Thank you.
